iOS/UZTickets/src/scan/UZTScanViewController.h

```diff
@@ -0,0 +1,33 @@
+//
+//  UZTScanViewController.h
+//  UZTickets
+//
+//  Created by nekoi on 6/1/15.
+//  Copyright (c) 2015 DAloG ltd. All rights reserved.
+//
+
+@import UIKit;
+
+@protocol UZTScanViewModelProtocol
+
+- (void)didStartInitializingScanView;
+- (void)didInitializeScanView;
+- (void)didReadQRCode:(NSString*)code;
+
+@end
```

Comment:
I've started from protocol for ViewModel, not from concrete class, therefore protocol is there.
Protocol is for different usecases of this exact scan viewcontroller. This way we could use it with different viewmodels.